base/logging.c

```diff
@@ -510,7 +510,7 @@ gvm_log_func (const char *log_domain, GLogLevelFlags log_level,
   /* Make the tmp pointer (for iteration) point to the format string. */
   tmp = prepend_format;
 
-  while (*tmp != '\0')
```

Comment:
I guess there should rather be a NULL check at line 483 like there is at line 441, to prevent prepend_format from ever being NULL.

Reply:
I don't think it's good to let tmp be NULL because then prepend_buf ends up "" instead of the default "%t %s %p - ".